client/src/containers/HomepageContainer/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import MenuItem from 'material-ui/MenuItem';
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card';
```

Comment:
A general rule of thumb is to put these on multiple lines once it has gone past two or three. 

Ex: 
```
import {
  Card,
  CardActions,
  CardHeader,
}
```